cmd/minikube/cmd/config/config_test.go

```diff
@@ -41,12 +41,14 @@ var configTestCases = []configTestCase{
     "ReminderWaitPeriodInHours": 99,
     "cpus": 4,
     "disk-size": "20g",
+    "iso-url": "http://foo.bar/minishift-centos.iso",
     "log_dir": "/etc/hosts",
     "show-libmachine-logs": true,
     "v": 5,
     "vm-driver": "kvm"
 }`,
 		config: map[string]interface{}{
+			"iso-url": "http://foo.bar/minishift-centos.iso",
```

Comment:
Good with tests, but they only seem to exercise the default case. What I am interested in as well, is whether I can have other settings in the config, eg _foo_ which basically get ignored then. The set/get functionality seems to just deal with _MinikubeConfig_. Can there be other data? If not, does this code throw an error? What if there is a mis-typed flag? Can we extend these tests a bit to cover some border cases? This will be a learning exercise as  well, since we might want to store other data in the config file which goes beyond _MinikubeConfig_. I would like to know whether this is possible. Also we need to better understand what happens, if the user manually edits these files and maybe introduces typos.
Also some tests around providing a config with not all settings in the config would be nice. One would then assert that default values apply.